common/source/docs/common-ltm-telemetry.rst

```diff
@@ -53,7 +54,10 @@ Parameter **SERIALX_PROTOCOL** has to be set to value *25* to enable LTM output
 
 .. note::
 
-    Don't forget to set baud rate to appropriate value. If you are using FSK modems, baud rate value depends on capability of modems.
+    Don't forget to set your LTM telemetry port's baud rate to an appropriate value. If you are using FSK modems, baud rate value depends on capability of modems. 
+    
+    Additionally, not all of your flight controller's serial ports support the lowest possible baud rate setting of 1 ( = 1200 baud). On STM32F4xx and F7xx based autopilots, these typically are USARTs 1 and 6 that run on the full system clock. If set to 1200 baud, these ports will default to 19200 baud instead. If you have issues getting correct LTM telemetry output at 1200 baud, try using a serial port other that USART 1 or 6. Check your flightcontroller's hardware page for further reference on UART to SERIALn mapping.
```

Comment:
some boards cannot do 1200 baud on any uarts, for example with current H7 config no uarts can do 1200. All can do 2400 baud